ServerCore/Pages/Puzzles/Feedback.cshtml

```diff
@@ -8,11 +8,29 @@
     ViewData["PlayRoute"] = "../Puzzles/SubmitFeedback";
     ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
     Layout = "_puzzleManagementLayout.cshtml";
-    var id = ViewContext.RouteData.Values["id"];
+
+    var sumFun = 0.0;
+    var sumDiff = 0.0;
+    var count = 0;
+    foreach (var item in Model.Feedbacks) {
+        sumFun += item.Fun;
+        sumDiff += item.Difficulty;
+        count++;
+    }
+    sumFun /= count;
+    sumDiff /= count;
 }
```

Comment:
This feels a bit out of place here - is this something that should be in the model instead? I'm not sure what the performance implications but usually we do the calculation in the model instead. Any reason you chose to have it here?